src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -353,4 +359,45 @@ public void Dispose()
 
         #endregion
     }
+
+    /// <summary>
+    /// Current this EventSource is only here so that the Debugger can inject code using Function evaluation
+    /// We may add actual logging requests as well at some point.   This can be removed if the debugger  no 
+    /// longer needs it.   
+    /// </summary>
+    [EventSource(Name = "Microsoft-Diagnostics-DiagnosticSource")]
+    class DiagnosticSourceEventSource : EventSource
```

Comment:
Nit: Assuming this type is meant to be internal rather than private to DiagnosticListener, I think it'd be good to separate it out into its own file.  Also, we try to be explicit about visibility, so it'd be good to expicitly state ```internal``` here.